Description: Correct use of unsupported NL macro in manpages
 opa manpages are shipped with with NL macro usage instead BR.
 This patch adds build logic to modify thqt macro in all opa manpages.
 .
 . opa (10.5.0.0-139-1ifs) UNRELEASED; urgency=medium
 .
 .  * Import opa 10.5.0.0-139 from IFS 10.5.0.0.153 upstream.
 
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Copyright: 2017 System Fabric Works, Inc. All Rights Reserved.
Forwarded: no
Last-Update: <2017-09-29>

---

--- a/IbaTools/man/Makefile
+++ b/IbaTools/man/Makefile
@@ -204,6 +204,8 @@
 # install for stage phase
 #include $(TL_DIR)/Makerules/Maketargets.stage
 STAGE::
+	sed 's/^\.NL\s*$$/.BR/g' -i $(MAN_PAGE_1_FILES) $(MAN_PAGE_8_FILES)
+	sed 'N;/\n/s/^\(.SH\s*NAME\s*\)\n\([^\r]*\)/\1\n\2 \\- command line utility/;P;D' -i $(MAN_PAGE_1_FILES) $(MAN_PAGE_8_FILES)
 	$(VS)$(STAGE_INSTALL) $(STAGE_INSTALL_DIR_OPT) $(PROJ_STAGE_MAN_DIR)/man1 $(MAN_PAGE_1_FILES)
 	$(VS)$(STAGE_INSTALL) $(STAGE_INSTALL_DIR_OPT) $(PROJ_STAGE_MAN_DIR)/man8 $(MAN_PAGE_8_FILES)
 
